packages/order-backend/src/service.spec.ts

```diff
@@ -249,7 +245,14 @@ describe('Test Create Order', () => {
 
   it('Should not throw error with given expired lastValidPurchaseDateTime on checkLastValidPurchaseDateTime function call', async () => {
     const lastValidPurchaseDate = new Date()
-    console.log('Time now %s', JSON.stringify(lastValidPurchaseDate))
+
+    console.log('Time now timezone %s', JSON.stringify(lastValidPurchaseDate))
```

Comment:
Could these comments be slightly different so it would be apparent which is the local/utc without timezone conversion and which is in Helsinki timezone?